index.js

```diff
@@ -24,7 +24,7 @@ const main = async () => {
   if (!project) {
     try {
       project = await fetchProjectId();
-    } catch (err) {
+    } catch {
```

Comment:
is this a new convention? I'm confused about why we remove (err) here

Reply:
I can't recall if this was an automatic or manual fix; but I presume that it's removed as it's an unused variable.